content/integrations/AWS-Integration/AWS-CloudFormation-Installation.md

```diff
@@ -12,31 +12,24 @@ This installation method leverages a CloudFormation script which can be viewed [
 
 After you make a new AWS integration in CloudWisdom, the script populates a read-only IAM role in your AWS account and links it using the integration’s Account ID and External ID. Once created, it may take a few minutes for the status to be updated.
 
-## Create an AWS Integration in CloudWisdom
-1. Open CloudWisdom and navigate to Integrations > AWS.  
-2. Create a new AWS integration. Ensure that the below tabled properties are set according to your needs.
+## 1. Create an AWS Integration in CloudWisdom
+1. Open CloudWisdom and navigate to **Integrations** > **Amazon Web Services**.  
+   - If this is not your first AWS integration, select **+ Add Integration**.
+   ![add-integration](/images/AWS-CloudFormation-Installation/add-integration.png)
+3. Enable the **Cost Explorer API**.
+4. Select the following Detailed Billing Source: **Cost And Usage Report (Recommended)**.
+ - Once you have finished all setup on this page, see the [Cost and Usage Report steps](/integrations/aws-integration/aws-cur).
```

Comment:
I think it would be better for a user to potentially visit the CUR page right at this step as they may forget when they get to the end. They can skip it if they know what it is and/or know they are able to skip it.
```suggestion
 - See the [Cost and Usage Report steps](/integrations/aws-integration/aws-cur).
```